tests/testthat/test-ggsave.R

```diff
@@ -57,3 +57,17 @@ test_that("text converted to function", {
 test_that("if device is NULL, guess from extension", {
   expect_identical(body(plot_dev(NULL, "test.png"))[[1]], quote(grDevices::png))
 })
+
+
+# parse_dpi ---------------------------------------------------------------
+
+test_that("DPI string values are parsed correctly", {
+  expect_is(parse_dpi("print"), "numeric")
```

Comment:
Makes sense, updated with those changes. Looks like `expect_type` only allows a single type so had to specify `double` (as it is on my system) rather than something like `c("integer", "double")`. I'm assuming this is fine and it won't auto-cast to integer or something like that on some systems.